### ISSUE TITLE
Feature request: allow setting esbuild keepNames options

### ISSUE THREAD

**Issue by canac** [netlify/zip-it-and-ship-it#622]
**- Do you want to request a _feature_ or report a _bug_?**

Feature request

**- What is the current behavior?**

My app uses a module that checks the functions' `name` property during validation (https://github.com/graphql-compose/graphql-compose-mongoose/blob/e4b916bf4c7a2e8857345ac9476d4e76bc25e030/src/resolvers/findById.ts#L42-L44). By default, esbuild renames functions during bundling, which changes their `name` property, which causes that check to fail.

**- What is the expected behavior?**

There is [`--keep-names`](https://esbuild.github.io/api/#keep-names) option in esbuild that keeps function names from being changed, which is exactly what I need. Can the zip-it-and-ship-it esbuild bundler expose that option?

**Comment by natevw** [comment]
Yes, this would also enable working around this issue with older versions of node-fetch which e.g. googleapis still rely on: <https://github.com/node-fetch/node-fetch/issues/784#issuecomment-1014768204>
